public/js/front.js

```diff
@@ -490,7 +490,13 @@ const deleteCart = (id, quantity, price) => {
         dataTodelete,
         function(data, status) {
             // json("Data: " + data + "\nStatus: " + status);
-
+            function notify(type, title, text) {
```

Comment:
We should look for a way to move this out so we can use them somewhere else. Working on that